.ai/spec/what/ocpmcp.md

```
@@ -0,0 +1,75 @@
+# OpenShift MCP Server (ocp-mcp)
+
+Standalone HTTPS OpenShift MCP server operand managed by the `ocpmcp` package ([OLS-3526](https://redhat.atlassian.net/browse/OLS-3526)). Replaces the former app-server sidecar. Related: [OLS-3572](https://redhat.atlassian.net/browse/OLS-3572) (agentic handoff), [OLS-3594](https://redhat.atlassian.net/browse/OLS-3594) (deferred agentic auto-injection).
+
+## Architecture
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)